ui/components/ui/site-origin/site-origin.js

```diff
@@ -19,4 +38,7 @@ SiteOrigin.propTypes = {
   siteOrigin: PropTypes.string.isRequired,
   iconName: PropTypes.string,
   iconSrc: PropTypes.string,
+  className: PropTypes.string,
+  title: PropTypes.string,
+  chip: PropTypes.bool,
```

Comment:
Would be good to add some propType descriptions. Not exactly sure what title does? Was it just added to ensure compatibility [here](https://github.com/MetaMask/metamask-extension/blob/develop/ui/components/app/connected-sites-list/connected-sites-list.component.js#L38)? 

Maybe something like this:

```suggestion
/**
   * The origin of the site generally the URL
   */
  siteOrigin: PropTypes.string.isRequired,
  /**
   * Icon name is used for the fallback icon and will display the first letter of the string
   * when iconSrc errors or is not provided.
   */
  iconName: PropTypes.string,
  /**
   * The image src of the icon
   */
  iconSrc: PropTypes.string,
  /**
   * Additional className added to the root html element
   */
  className: PropTypes.string,
  /**
   * title specifies extra information and is added to a title attribute added to the root html element
   * It is most often shown as a tooltip text when the mouse moves over the element.
   */
  title: PropTypes.string,
  /**
   * Whether the SiteOrigin is displayed as a chip or plan text.
   * if false iconSrc and iconName props will not be used.
   */
  chip: PropTypes.bool,
```